grid2op/Opponent/OpponentWithConverter.py

```diff
@@ -0,0 +1,183 @@
+# Copyright (c) 2019-2020, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0.
+# If a copy of the Mozilla Public License, version 2.0 was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
+
+from abc import abstractmethod
+
+from grid2op.Converter import Converter
+from grid2op.Exceptions import Grid2OpException
+from grid2op.Opponent import BaseOpponent
+
+
+class OpponentWithConverter(BaseOpponent):
```

Comment:
it's not clear for me, this class exists or not now ?

Reply:
No it is deleted, it is just not pushed yet